src/scripts/main.js

```diff
@@ -1 +1,75 @@
 'use strict';
+
+function createDiv(divMessage, divStatus) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.className = divStatus;
+  div.innerText = divMessage;
+
+  document.body.appendChild(div);
+}
+
+document.addEventListener('contextmenu', (e) => {
+  e.preventDefault();
+});
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
```

Comment:
The first promise will always be rejected after 3 seconds if no click event occurs. Consider adding a mechanism to clear the timeout if the promise resolves to prevent unnecessary rejection.